.gitignore

```diff
@@ -28,4 +28,6 @@ hs_err_pid*
 build
 
 # Ignore JEdit temp file
-*~
\ No newline at end of file
+*~
+**/#*#
+/bin1/
```

Comment:
Can we remove this line? I'm not sure what it is coming from or what it corresponds to.